test_requirements/requirements_base.txt

```diff
@@ -26,4 +26,4 @@ sphinx
 sphinxcontrib-spelling
 pyflakes
 pyenchant
-
+mock>=1.3.0,<2.0.0
```

Comment:
I wouldn't call these tests useless, feel free to update them to real objects. It's not simple to figure out how to do so with your test layout in that module.